Tools/scripts/checkpyc.py

```diff
@@ -51,7 +51,7 @@ def main():
                     print('Bad MAGIC word in ".pyc" file', end=' ')
                     print(repr(name_c))
                     continue
-                mtime = get_long(mtime_str)
+                mtime = get_long(mtime_str) & 0xFFFF_FFFF
```

Comment:
This is actually a NOP, because `mtime_str` is just 4 bytes, that cannot encode anything larger than `0xFFFF_FFFF`

Reply:
thanks, I am not familiar with the operations on the bits.